components/namespace-lister/empty-base/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: namespace-lister
```

Comment:
```suggestion
```

Reply:
Kustomize needs something other than an empty config, otherwise it will exit with an error (and the corresponding application in argo will always be reporting an error).  Applying this change yields the following:
```
$ kustomize build components/namespace-lister/empty-base
Error: kustomization.yaml is empty
```

Note that this configuration won't actually make any resources, since the namespace generator does nothing without any resources to act on.